modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/confirm.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+export function confirm({message, onConfirm}) {
+	Liferay.Util.openModal({
```

Comment:
We can import `openModal` here, instead of using `Liferay` global.

Reply:
See "Additional comments"  on PR description

Reply:
Ah yes, sorry I missed this! You need to set utility up with `Liferay.Loader.require`, like [other utilities with React](https://github.com/liferay/liferay-portal/blob/master/modules%2Fapps%2Ffrontend-js%2Ffrontend-js-web%2Fsrc%2Fmain%2Fresources%2FMETA-INF%2Fresources%2Fliferay%2Fglobal.es.js#L240-L265).

Reply:
It's not a bug, it's a feature :)